src/Console/Output/ProcessOutputFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Console\Output;
+
+final class ProcessOutputFactory
+{
+    public static function create(string $outputType, OutputContext $context): ProcessOutputInterface
```

Comment:
no static methods pls.

static is as evil as global.
good luck for whoever later to introduce proper DIC or mocking ProcessOutputFactory as dependency.